### PR TITLE
Renaming TeamWilder -> NextDoorNature

### LINKED ISSUE #107: Use NextDoorNature, not TeamWilder
## Description
Change references of TeamWilder to NextDoorNature

TeamWilder is not a campaign. It is a way of thinking. Not to be used as a brand/in this way.
We need to get ‘NextDoorNature’ into the copy more. 

## Acceptance Criteria

- [ ] `find src/ -type f | xargs grep -i --color "team\\s*wilder"` should return nothing
- [ ] The correct wording for this is "Nextdoor Nature"
- [ ] @ivan-kocienski-gfsc draft and get @undividual to confirm

## Plan

- [ ] Scan for "team" in codebase
- [ ] Remove/change name
- [ ] rename local directory
- [ ] rename repo on GH (?)

### PR DESCRIPTION
**Pull request by ivan-kocienski-gfsc**
Fixes #107

## Description

Use correct project name for client

@geeksforsocialchange/developers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Team Wilder
+# Nextdoor Nature
 
 [Current release](https://team-wilder-proto.pages.dev/)
```
Comment (katjam) [inline README.md:3]: Are we going to change this too?

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "teamwilder",
+  "name": "nextdoornature",
```
Comment (katjam) [inline package.json:2]: I think this is technically the repo name which is teamwilder, but willing to be wrong on that.

File: src/I18n/En.elm [hunk 3]
```diff
@@ -119,10 +119,10 @@ enStrings key =
         -- Index page
         ---
         WelcomeP1 ->
-            "[cCc] TeamWilder is an initiative launched by the Wilidlife Trusts in 2022. (Read more here) The aim of TeeamWilder is to equip everyone with the skills and knowledge they need to make a positive difference for nature - in their own area, and beyond."
```
Comment (katjam) [inline src/I18n/En.elm:122]: lol Teeeam!

**Review by katjam — COMMENTED** [review]
Couple of questions / comments but probably not worth getting hung up on.

File: src/I18n/En.elm (hunk 3 above)
Comment (ivan-kocienski-gfsc) [reply src/I18n/En.elm:122]: I also spotted that. happens to the best of us

File: README.md (hunk 1 above)
Comment (ivan-kocienski-gfsc) [reply README.md:3]: Sure why not? Perhaps some wording about the relationship may be in order.

Comment (ivan-kocienski-gfsc) [reply README.md:5]: I put in a small comment explaining the relationship to WT TeamWilder

File: package.json (hunk 2 above)
Comment (katjam) [reply package.json:2]: It's meant to be the name of the package, like if you publish on npm - so it should match the repo but it won't affect anything,

**Review by katjam — APPROVED** [review]
Looks fine!

**Pull request merged**